### PR TITLE
Flagsmith - added dashboard

### PR DESCRIPTION
**Pull request by dabeeeenster**
### What does this PR do?

Adds a Flagsmith dashboard to the Flagsmith integration

### Motivation

We needed a dashboard!
### Review checklist

- [X] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [X] Feature or bugfix has tests
- [X] Git history is clean
- [X] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

None

### CODE REVIEW

**Review by nmuesch — CHANGES_REQUESTED** [review]
Hey! Thanks for adding the dashboard. The validation is failing because the Dashboard payload needs to use the App UUID defined in your `manifest.json` file instead of the local app uuid from your org. 

Could you update that and also please format the JSON using an online or local json formatter? It'll make this diff and future diffs smaller and quicker to review!

**Comment by dabeeeenster** [comment]
OK I'm seeing green how thanks!

**Review by nmuesch — APPROVED** [review]

File: flagsmith/assets/dashboards/flagsmith-dashboard.json [hunk 1]
```diff
@@ -0,0 +1,92 @@
+{
+  "title": "Flagsmith Dashboard",
+  "description": null,
```
Comment (eho1307) [inline flagsmith/assets/dashboards/flagsmith-dashboard.json:3]: Could you add a short description describing what's available on the dashboard?

File: flagsmith/assets/dashboards/flagsmith-dashboard.json [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+        "y": 0,
+        "width": 7,
+        "height": 3
+      }
+    },
+    {
+      "id": 2359149605302098,
+      "definition": {
+        "type": "note",
+        "content": "# Welcome to the Flagsmith Dashboard\n\nYou can integrate Flagsmith with Datadog in three ways:\n\n- [1. Integrate Flagsmith into your Datadog Dashboard](https://docs.flagsmith.com/integrations/apm/datadog#1-integrate-flagsmith-into-your-datadog-dashboard)\n- [2. Send Flag Change events to Datadog](https://docs.flagsmith.com/integrations/apm/datadog#2-send-flag-change-events-to-datadog)\n- [3. Integrate with the DataDog RUM](https://docs.flagsmith.com/integrations/apm/datadog#3-integrate-with-the-datadog-rum)\n\n",
```
Comment (alai97) [inline flagsmith/assets/dashboards/flagsmith-dashboard.json:27]:
```suggestion
        "content": "# Welcome to the Flagsmith Dashboard\n\nYou can integrate Flagsmith with Datadog in three ways:\n\n- [1. Integrate Flagsmith into your Datadog Dashboard](https://docs.flagsmith.com/integrations/apm/datadog#1-integrate-flagsmith-into-your-datadog-dashboard)\n- [2. Send Flag Change events to Datadog](https://docs.flagsmith.com/integrations/apm/datadog#2-send-flag-change-events-to-datadog)\n- [3. Integrate with Datadog RUM](https://docs.flagsmith.com/integrations/apm/datadog#3-integrate-with-the-datadog-rum)\n\n",
```

**Review by alai97 — COMMENTED** [review]
Looks good, please add a description!

File: flagsmith/assets/dashboards/flagsmith-dashboard.json (hunk 1 above)
Comment (dabeeeenster) [reply flagsmith/assets/dashboards/flagsmith-dashboard.json]: OK that's added thanks

**Comment by dabeeeenster** [comment]
I think this is everything - please @ me if you need more edits, thanks.

**Review by eho1307 — APPROVED** [review]
Thanks!

**Pull request merged**